### DISCUSSION TITLE
Pregabalin + SSRIs

### DISCUSSION THREAD

**Discussion by zelixir25** [TripSit/drugs#80] [Questions]
This combination is unlikely to cause any unpleasant or dangerous side effects.

Arnold, L. M., Sarzi-Puttini, P., Arsenault, P., Khan, T., Brown, P. B., Clair, A., Scavone, J. M., Driscoll, J., Landen, J., & Pauer, L. (2015). Efficacy and safety of pregabalin in patients with fibromyalgia and comorbid depression taking concurrent antidepressant medication: A randomized, placebo-controlled study. The Journal of Rheumatology, 42(7), 1237–1244. https://doi.org/10.3899/jrheum.141196

Baldwin, D. S., den Boer, J. A., Lyndon, G., Emir, B., Schweizer, E., & Haswell, H. (2015). Efficacy and safety of pregabalin in generalised anxiety disorder: A critical review of the literature. Journal of Psychopharmacology, 29(10), 1047–1060. https://doi.org/10.1177/0269881115598411

**Comment by utaninja** [comment]
I am hesitant to push this until we have a pharmacologist help advice on Pregabalin seizure concerns. I'd be worried that this may be of higher risk when taking high doses of Pregabalin with SSRI's. I'll be reaching out to two different people today, hopefully should not take so long. I would also remember these studies are for medical dosages and should be factored in, though there is crossover of medical and recreational dosages.

Are you able to highlight what this study is demonstrating for this interaction "Baldwin, D. S., den Boer, J. A., Lyndon, G., Emir, B., Schweizer, E., & Haswell, H. (2015). Efficacy and safety of pregabalin in generalised anxiety disorder: A critical review of the literature. Journal of Psychopharmacology, 29(10), 1047–1060. https://doi.org/10.1177/0269881115598411"?

**Reply by zelixir25** [reply]
Baldwin, et al. (2015) talks a lot about adverse events in studies using pregabalin for GAD and some of the studies they selected for their literature review studied pregabalin as adjunctive therapy with SSRIs and SNRIs. They mainly cited Rickels, et al. (2012), which did not show any significant increase in AEs in patients who received pregabalin in addition to their SSRIs or SNRIs.

I also found Alvarez, et al. (2015) which used pregabalin as an adjunct to SSRIs and SNRIs and did not report any major adverse events, though that was not their primary concern. I imagine they would have reported something had there been major health effects such as seizures.

In both these studies, patients were given flexible dosing ranging from 75mg (Alvarez, et al., 2015) or 100mg/day (Rickels, et al., 2012) to 600mg/day for both studies.

Álvarez, E., Olivares, J. M., Carrasco, J. L., López-Gómez, V., & Rejas, J. (2015). Clinical and economic outcomes of adjunctive therapy with pregabalin or usual care in generalized anxiety disorder patients with partial response to selective serotonin reuptake inhibitors: Annals of General Psychiatry. Annals of General Psychiatry, 14(1), 18–43. https://doi.org/10.1186/s12991-014-0040-0

Rickels, K., Shiovitz, T. M., Ramey, T. S., Weaver, J. J., Knapp, L. E., & Miceli, J. J. (2012). Adjunctive therapy with pregabalin in generalized anxiety disorder patients with partial response to SSRI or SNRI treatment. International Clinical Psychopharmacology, 27(3), 142. https://doi.org/10.1097/YIC.0b013e328350b133

**Reply by utaninja** [reply]
People will go greatly above 600mg for recreational dosages. 600mg is used commonly as a number of which past this is when there is serious concern, although to note I have not found a basis for this number. Regardless, the higher the dosage is certainly has a correlation with seizure reports of pregabalin usage. These are very helpful studies but I am unsure of the mix in this regard.

**Reply by zelixir25** [reply]
Could always go with "Caution: At doses up to 600mg Pregabalin, this combination seems well-tolerated. However, higher doses of Pregabalin may increase seizure risk." or something like that. I think it's best to outline what we know and what we don't know here. Of course, digging for more studies that use higher doses would be helpful too if there are any out there.

**Reply by utaninja** [reply]
We haven't proven that it may increase seizure risk. I'm cautious of putting such a claim in interactions without having any sort of explanation of the logical intuition to it or medical sources. There seems to be a decent amount of anecdotal reports to it but not much more than that. I've asked psychcombo for help in this regard and am waiting to hear back.

**Reply by zelixir25** [reply]
I understand that scientific sources are best but if there's several anecdotal reports I don't see the problem in mentioning that either. "While scientific literature has not documented an increased seizure risk for this combination with doses up to 600mg Pregabalin, there are several anecdotal reports of seizures from individuals using higher doses." the point here imo is to just present what we know and let people make up their mind. I think we would be remiss to not mention it.

**Reply by utaninja** [reply]
If we take this approach, we might find issues when this comes to other substances that have risks of seizures. Do you think we should say pregabalin raises risk for every drug combined with it that also has seizure risks? I would just like to make sure there's consistency and as accurate information as possible. I'm not opposed with going there's a great amount of anectodal evidence but we need to clarify what sort of criteria we are using.

**Comment by utaninja** [comment]
Pregabalin + SSRIs
Unsafe
There is a significant risk of seizures from this combination. The combination may increase each other’s side effects such as dizziness and drowsiness. Medical dosages do not seem of concern for the general population.

---

As discussed in https://github.com/TripSit/drugs/discussions/40, pregabalin is reported to have seizure concerns, in particular at higher dosages and to at-risk individuals. SSRIs seem to have a reported concern of seizures as well, in particular to at-risk individuals and in drug combinations. I was unable to find sources discussing seizure concerns between pregabalin and SSRIs. Notably, pregabalin already has limited reports on its seizure concerns, which would possibly help explain the lack of discussion or reports between the substances. It seems reasonable to mark the interaction as unsafe due to interactions on the chart presuming pregabalin is at recreational (600mg+ doses). I’d appreciate a second look at this combination.

For ease of use, I’ve attached relevant studies I found below, and for sources on pregabalin’s seizures please see the cited discussion above.
[Relative Toxicity of Selective Serotonin Reuptake Inhibitors (SSRIs) in Overdose](https://doi.org/10.1081/CLT-120037428)
[Antidepressant Drugs for Seizures and Epilepsy: Where do we Stand?](https://doi.org/10.2174/1570159X20666220627160048)
[Antidepressant use and risk of epilepsy and seizures in people aged 20 to 64 years: cohort study using a primary care database](https://doi.org/10.1186/s12888-015-0701-9)
[Pregabalin: a review of its use in adults with generalized anxiety disorder](https://doi.org/10.1007/s40263-014-0192-0)
Prescriber's Guide: Stahl's Essential Psychopharmacology discusses and notes SSRI seizure concerns
[Efficacy and Safety of Pregabalin in Patients with Fibromyalgia and Comorbid Depression Taking Concurrent Antidepressant Medication: A Randomized, Placebo-controlled Study](https://doi.org/10.3899/jrheum.141196)
[Comparative Efficacy of Newer Antidepressants in Combination with Pregabalin for Fibromyalgia Syndrome: A Controlled, Randomized Study](https://doi.org/10.1111/papr.12409)
[Adjunctive therapy with pregabalin in generalized anxiety disorder patients with partial response to SSRI or SNRI treatment](https://doi.org/10.1097/yic.0b013e328350b133)
